apps/people-app/webapp/src/view/master-data/sections/danger-section/DeleteCurrent.tsx

```
@@ -0,0 +1,147 @@
+  }, [confirmOpen, pendingCloseAfterSpinner, showSpinner, onDeleteSuccessComplete]);
+
+  const handleConfirm = async (reason: string) => {
+    setConfirming(true);
+    try {
+      await onDelete(reason);
+      if (!isMountedRef.current) return;
+      setPendingCloseAfterSpinner(true);
+    } catch (e) {
+      console.error(`Delete ${nodeTypeLabel} failed`, e);
+    } finally {
+      if (!isMountedRef.current) return;
```

Biome (2.4.11)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)